src/main/java/duke/command/Find.java

```diff
@@ -0,0 +1,32 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.DukeException;
+import duke.exception.InvalidFormatException;
+import duke.tasks.Task;
+
+public class Find extends Command {
```

Comment:
I think there should be a javaDocs for this class

Reply:
Will add in!